public/css/list/item-list.less

```diff
@@ -170,6 +171,51 @@
   }
 }
 
+.item-list {
+  .icon-image {
+    width:  3em;
+    height: 3em;
+    align-self: center;
+    text-align: center;
+
+    &.inline {
+      display: inline-block;
+      height: 1.5em;
+      width: 1.5em;
+    }
+
+    img {
+      vertical-align: middle;
+      max-height: 100%;
+      max-width: 100%;
+      height: auto;
+      width: auto;
+    }
+  }
+
+  &.minimal {
+    .icon-image {
+      height: 2em;
+      width: 2em;
+      line-height: 2;
+      align-self: center;
+      margin-top: 0;
+
+      img {
+        vertical-align: middle;
+      }
+    }
+  }
+
+  .title {
+    .icon-image {
+      display: inline-block;
+      width: 1.5em;
+      height: 1.5em;
+    }
+  }
```

Comment:
Isn't this a remnant from the host icon in the service list? Please remove.

Reply:
✅